.infrastructure/98_variables.tf

```diff
@@ -63,4 +57,10 @@ variable "publish_cloudfront_functions" {
   type        = bool
   description = "Defines if cloudfront functions should be published"
   default     = false
+}
+
+variable "dns_domain_name" {
+  description = "DNS domain for the b2b portals"
+  type        = map(any)
```

Comment:
why a map as type of the DNS name?

Reply:
Because with a module "zone" the dns name not is a String but a map type: 
https://github.com/terraform-aws-modules/terraform-aws-route53/blob/master/modules/zones/README.md#input_zones

Reply:
ok, with new commit: source = "git::github.com/terraform-aws-modules/terraform-aws-route53.git?ref=fb53f9541723161ff39374add047dd8e60441e10" # v2.11.0